_changelog/entries/2021/2021-05-06-shopify-v1-remove-build-fields-duplicates.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Shopify (v1) bug-fix: Remove duplicates of `build` fields"
+content-type: "changelog-entry"
+date: 2021-05-06
+entry-type: bug-fix
+entry-category: integration
+connection-id: "shopify"
+connection-version: "1"
+pull-request: "https://github.com/singer-io/tap-shopify/pull/103"
+---
+{{ site.data.changelog.metadata.single-integration | flatify }}
+
+Due to the case sensitivy of fields during the {{ this-connection.display_name }} replication job, Stitch failed to recognize that `build` and `Build` were the same field and would produce duplicate records. This bug-fix scans for, and removes those duplicates. 
```

Comment:
I feel like this is only half the picture here. Stitch isn't actually removing duplicates during Extraction - the integration is canonicalizing `build` and `Build` since they're essentially the same field.

This means that whenever the integration sees `Build`, it'll transform the field name to `build`. This results in all data for `build` and `Build` going into a single column, `build`.